Automattic/simplenote-android
# Issue/757 note widget resizable

### Fix
Update the widget to allow the view to be expanded to more than 1x1 to close #757.  See the animation and screenshot below for illustration.

<table>
<tr>
<th>Resize</th>
<th>Sizes</th>
</tr>
<tr>
<td><img src="https://user-images.githubusercontent.com/3827611/61919994-2d423b00-af15-11e9-9a92-23c5251b5b39.gif"></td>
<td><img src="https://user-images.githubusercontent.com/3827611/61919997-2f0bfe80-af15-11e9-917b-bf8074c519b3.png"></td>
</tr>
</table>

### Test
These steps assume the AOSP launcher (i.e. Nexus/Pixel launcher).  The steps to add the widget to the home screen may be slightly different based on the launcher used.  Other steps should be the same regardless of launcher.
1. Long-press home screen.
2. Tap ***Widgets*** option.
3. Scroll to ***Simplenote*** widget.
4. Notice ***To-Do List*** preview image.
5. Long-press ***Note*** widget.
6. Place ***Note*** widget on home screen.
7. Notice ***Select Note*** dialog with note list.
8. Tap note in list.
9. Notice note title shown in widget.
10. Long-press widget.
11. Notice circular handles are shown.
12. Drag circular handles horizontally and vertically.
13. Notice note title and content are shown in widget when large enough.

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in 508b7c4a with:
> Added resizing the note widget to display title and content
